packages/app/src/components/Sliders/Slider.module.css

```diff
@@ -29,6 +42,15 @@
     height: 14px;
     border-radius: 100vh;
     cursor: pointer;
+
+    background: linear-gradient(
+      to bottom,
+      oklab(from var(--color) calc(l + 0.2) a b),
```

Comment:
Is there some meaning behind 0.2? Perhaps avoid magic number?

Reply:
Not really, its just a fine tuning. If we ever want to reuse more of this, we should extract into a css variable